### DISCUSSION TITLE
Can you do this?

### DISCUSSION THREAD

**Discussion by RedRavenYouTube** [LBPUnion/ProjectLighthouse#637] [Q&A] [answered]
Can you make LBPU available on LBPPSP? Or is it too different to the mainline LBP games.

**Comment by m88youngling** [comment, marked as answer]
LBP Union and its values can go wherever we want! We take our community-first mindset everywhere we go in life, and make the changes we want to see in our world together.

What you probably meant to ask is if _Project Lighthouse_ will support LBP PSP. The answer is probably not, unless someone finds a way to emulate PlayStation Network for PSP. Without PSN server endpoints, it's just not possible.

**Reply by RedRavenYouTube** [reply]
Will it work if you use the game on a PSVita?

**Reply by m88youngling** [reply]
LBP Vita is supported. Please see this article https://www.lbpunion.com/post/project-lighthouse-littlebigplanet-private-servers

LBP PSP running in Adrenalin (psp emulator for PS Vita) wouldn't allow for online play either, because the emulator can't connect to nonexistent PlayStation Network endpoints.

**Reply by sudokoko** [reply]
And even if we wanted to actually attempt and revive those broken endpoints, probably by means of custom servers, Sony probably would not like that, at all.

**Reply by TorutheRedFox** [reply]
> And even if we wanted to actually attempt and revive those broken endpoints, probably by means of custom servers, Sony probably would not like that, at all.

RPCN does exactly this and Sony doesn't care

**Reply by KairosEasyBro** [reply]
> > And even if we wanted to actually attempt and revive those broken endpoints, probably by means of custom servers, Sony probably would not like that, at all.
> 
> RPCN does exactly this and Sony doesn't care

The thing is, RPCS3 is an emulator so RPCN developers didn't need to reverse engineer everything to connect to PSN on real hardware, but to make it work on PSP, we need the keys and all of that stuff and that's a no-no (mostly because PSN PSP is still PSN and some security stuff is probably shared with PS3 and up).

**Reply by schm1dtmac** [reply]
PSP PSN is... still up though? Why would we need broken endpoint revival or PSN emulation, I've still got my PSP 1000 logged into PSN just fine and have been using that to access LittleBigRefresh. (LBR is shutting down though unfortunately, hence my sudden interest in Lighthouse PSP support)

**Comment by RedRavenYouTube** [comment]
What I am also saying is can you use a PSVITA and download LBP PSP on that vita and use a custom version of the vita's PSN?

**Reply by indo2042** [reply]
If you can somehow manage to find/make a version of PSN for the PSP that can compensate for the missing endpoints, that will probably work.

**Reply by TorutheRedFox** [reply]
> What I am also saying is can you use a PSVITA and download LBP PSP on that vita and use a custom version of the vita's PSN?

the Vita just emulates a PSP with a slightly cut down version of the firmware (basically no VSH and bloat you'd only be able to run through it), so I doubt that'd work
